types/backgrid/index.d.ts

```diff
@@ -101,9 +101,9 @@ declare namespace Backgrid {
         insertRow(model: Backbone.Model, collection: Backbone.Collection<Backbone.Model>, options: any);
         moveToNextCell(model: Backbone.Model, cell: Column, command: Command);
         refresh(): Body;
```

Comment:
Not `refresh`?

Reply:
I wanted to make changes in backbone that affect the minimum number of other packages. In another PR may be.